rpcserver.go

```
@@ -4131,6 +4132,7 @@ func createRPCOpenChannel(r *rpcServer, dbChannel *channeldb.OpenChannel,
 
 	// Fetch the set of aliases for the channel.
 	channelAliases := r.server.aliasMgr.GetAliases(dbScid)
+	peerScidAlias, _ := r.server.aliasMgr.GetPeerAlias(chanID)
```

An ignored error should at least have a comment associated with it.